src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/client/ApOASysContextApiClient.kt

```diff
@@ -38,10 +37,6 @@ class ApOASysContextApiClient(
     path = "/needs-details/$crn"
   }
 
-  fun getHealthDetails(crn: String) = getRequest<HealthDetails> {
-    path = "/health-details/$crn"
```

Comment:
should we: 

- include a reference to our wiremock example: https://github.com/ministryofjustice/hmpps-approved-premises-tools/blob/main/wiremock/mappings/ApAndOasys_GetHeath.json   ?
- create an accompanying PR to remove the wiremock example stub from API-Tools?

Reply:
Hi Ed, good catch, I hadn't considered wiremock in ap-tools. I'm not sure that we need to explicitly ref it in the commit, but i'll create a PR to remove it. I've also found a wiremock config in this project, so i've removed that in this commit too.